src/documents/ChangeApplierAPI.md

```diff
@@ -244,7 +245,7 @@ The values of built-in sources supported as values in `excludeSource` and `inclu
     </tbody>
 </table>
 
-##### Example featuring built-in change source filtering:
+##### Example featuring built-in change source filtering: ####
```

Comment:
some of the headers have imbalanced #s. for example this one has 5 on the left and 4 on the right. I think there are a few other cases like this as well.

Reply:
So, we should discuss.  This is one of three valid styles, and we use all three styles in this package.  The only check I added was to force them to be consistent within a single file.   If we want to enforce "closed" headers, I can flip that switch and fix the outliers, but I'd estimate an hour or two of cleanup, as there were many examples of both the "open" style and the "header followed by a line of equals" style.

Reply:
It would be nice to have consistency across files, and thank you for making them at least consistent within the file. I'm not sure which style we want to settle on, but we may want to consider a "style guide" at some point. Regarding my comment above though, I was really just mentioning that with the closed style (assuming that the markdown above is in closed style), it'd be good to have the same number of "#" on either side. Otherwise it may be confusing about which level it actually is.

Reply:
*facepalm*.  I was looking at what I'd removed.  You're absolutely right, it does appear unbalanced.  However, having them balance is not covered by the markdown linter, I checked with both "consistent" and "atx closed".

I went to report a feature request, but in researching for that I discovered that balancing the closing headers is in fact not part of the CommonMark spec by design:

http://spec.commonmark.org/0.28/#example-42

This might be an argument in favour of using "atx" headers (no closing), which less typing and which can be linted.  For now, I just "balanced" the closing header until we can discuss.

Speaking of which, thanks for accepting the invite to chat on Monday.  I will mention our upcoming chat at the architecture meeting, IMO a draft "style guide" would be a good next step once we've resolved the current issues and talking about some of the style concerns.  I was thinking we could make a style guide part of the documentation for the small plugin we use to standardise our rules, similar to what we've done with ESLint.